src/main/kotlin/no/nav/familie/ef/personhendelse/configuration/KafkaConfig.kt

```diff
@@ -14,11 +14,11 @@ import org.springframework.kafka.core.DefaultKafkaConsumerFactory
 class KafkaConfig {
 
     @Bean
-    fun kafkaPersonhendelseListenerContainerFactory(properties: KafkaProperties, kafkaErrorHandler: KafkaErrorHandler)
-            : ConcurrentKafkaListenerContainerFactory<Long, Personhendelse> {
-        val factory = ConcurrentKafkaListenerContainerFactory<Long, Personhendelse>()
-        factory.consumerFactory = DefaultKafkaConsumerFactory(properties.buildConsumerProperties())
-        factory.setErrorHandler(kafkaErrorHandler)
-        return factory
-    }
-}
\ No newline at end of file
+    fun kafkaPersonhendelseListenerContainerFactory(properties: KafkaProperties, kafkaErrorHandler: KafkaErrorHandler):
+        ConcurrentKafkaListenerContainerFactory<Long, Personhendelse> {
```

Comment:
Hmm, litt usikker på dette med `): <br> <return-type>`

Reply:
Enig. Ktlint gjorde dette verre synes jeg. Grunnen til at den gjør det er fordi linja blir for lang.